CONTRIBUTING_TO_OPENSHIFT_JENKINS_IMAGE_AND_PLUGINS.md

```diff
@@ -49,6 +49,10 @@ An example flow when running in an OpenShift pod:
     ```
     git clone https://github.com/openshift/<plugin in question>-plugin.git
     ```
+1. Enable the provided git hooks
```

Comment:
I would elaborate here what these git hooks achieve re: your PR description here and the PR description in #1374

Reply:
Also make sure you highlight the distinction we noted in planning today:  that what you are doing to facilitate the CPaas flow in not way impacts what we have to do for the traditional ART flow while we still have Jenkins images in the payload.

Reply:
I added that into the README.md and set a link to it in CONTRIBUTING.md